homeassistant/components/overkiz/__init__.py

```diff
@@ -0,0 +1,123 @@
+"""The Overkiz (by Somfy) integration."""
+import logging
+from typing import NamedTuple
+
+from aiohttp import ClientError, ServerDisconnectedError
+from pyoverkiz.client import OverkizClient
+from pyoverkiz.const import SUPPORTED_SERVERS
+from pyoverkiz.exceptions import (
+    BadCredentialsException,
+    MaintenanceException,
+    TooManyRequestsException,
+)
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+
+from .const import (
+    CONF_HUB,
+    DOMAIN,
+    PLATFORMS,
+    UPDATE_INTERVAL,
+    UPDATE_INTERVAL_ALL_ASSUMED_STATE,
+)
+from .coordinator import OverkizDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class HomeAssistantOverkizData(NamedTuple):
```

Comment:
In the future, a dataclass would be preferred since our minimum python version is 3.9 now

https://github.com/home-assistant/core/blob/dev/homeassistant/components/lookin/models.py#L12

Reply:
Fixed